services/hangar/hangar-downloads.service.js

```diff
@@ -0,0 +1,34 @@
+import { pathParams } from '../index.js'
+import { renderDownloadsBadge } from '../downloads.js'
+import { BaseHangarService, description } from './hangar-base.js'
+
+export default class HangarDownloads extends BaseHangarService {
+  static category = 'downloads'
+
+  static route = {
+    base: 'hangar/downloads',
```

Comment:
If this is total downloads (all time), URL should be `/dt` - see https://github.com/badges/shields/blob/master/doc/badge-urls.md